src/Context/index.js

```diff
@@ -1,7 +1,22 @@
-import React, { createContext } from "react";
+import React, { createContext, useState } from "react";
 
 export const TodoContext = createContext({});
 
 export const TodoProvider = ({ children }) => {
-  return <TodoContext.Provider value={{}}>{children}</TodoContext.Provider>;
+  const [list, setList] = useState([]);
+
+  function handleAddItem(task) {
+    setList([...list, { mainTask: task }]);
+  }
+
+  function handleDelete(item) {
+    const newList = list.filter((task) => task !== item);
```

Comment:
Para remover utilize o ID
```suggestion
    setList(list.filter((task) => task.id !== id));
```